src/internal/observable/throwError.ts

```diff
@@ -0,0 +1,64 @@
+import { Observable } from '../Observable';
+import { IScheduler } from '../Scheduler';
+import { Subscriber } from '../Subscriber';
+
+/**
+ * Creates an Observable that emits no items to the Observer and immediately
+ * emits an error notification.
+ *
+ * <span class="informal">Just emits 'error', and nothing else.
+ * </span>
+ *
+ * <img src="./img/throw.png" width="100%">
+ *
+ * This static operator is useful for creating a simple Observable that only
+ * emits the error notification. It can be used for composing with other
+ * Observables, such as in a {@link mergeMap}.
+ *
+ * @example <caption>Emit the number 7, then emit an error.</caption>
+ * import { throwError, concat, of } from 'rxjs/create';
+ *
+ * const result = concat(of(7), throwError(new Error('oops!')));
+ * result.subscribe(x => console.log(x), e => console.error(e));
+ *
+ * @example <caption>Map and flatten numbers to the sequence 'a', 'b', 'c', but throw an error for 13</caption>
+ * import { throwError, interval, of } from 'rxjs/create';
+ * import { mergeMap } from 'rxjs/operators';
+ *
+ * interval(1000).pipe(
+ *   mergeMap(x => x === 13 ?
+ *     throwError('Thirteens are bad') :
+ *     of('a', 'b', 'c')
+ *   )
+ * ).subscribe(x => console.log(x), e => console.error(e));
+ *
+ * @see {@link create}
+ * @see {@link empty}
+ * @see {@link never}
+ * @see {@link of}
+ *
+ * @param {any} error The particular Error to pass to the error notification.
+ * @param {Scheduler} [scheduler] A {@link IScheduler} to use for scheduling
+ * the emission of the error notification.
+ * @return {Observable} An error Observable: emits only the error notification
+ * using the given error argument.
+ * @static true
+ * @name throw
+ * @owner Observable
+ */
+export function throwError(error: any, scheduler?: IScheduler): Observable<never> {
```

Comment:
To avoid a big break you could add a `type` `export type ErrorObservable = Observable<never>`, that will avoid an errors if someone has an explicit type reference (right-hand-side), but not if they have a value reference.

Reply:
That's an interesting idea! I'll have to think about it. I could probably do it elsewhere as well. 🤔